pkg/logging/logging.go

```diff
@@ -29,7 +29,7 @@ const (
 )
 
 var (
-	logger = runtimelog.NewDelegatingLogger(runtimelog.NullLogger{})
+	logger = runtimelog.Log
```

Comment:
What is the benefit of this change? I don't fully understand it in the context of the migration yet